mycore-base/src/main/java/org/mycore/frontend/fileupload/MCRUploadHelper.java

```diff
@@ -46,6 +49,16 @@ public abstract class MCRUploadHelper {
 
     private static final Pattern PATH_SEPERATOR = Pattern.compile(Pattern.quote(File.separator.replace('\\', '/')));
 
+    public static final Predicate<String> FILE_NAME_PREDICATE = initializePredicate();
+
+    public static final String FILE_NAME_PATTERN = "MCR.FileUpload.FileNamePattern";
```

Comment:
misleading field name FILE_NAME_PATTERN should contain a file name pattern. The value of this field is a property name!